src/expr.cpp

```diff
@@ -1494,15 +1522,15 @@ std::string UnaryExpr::GetString() const {
 }
 
 void UnaryExpr::Print(Indent &indent) const {
-    if (!expr || !GetType()) {
```

Comment:
It is interesting if this change fixes the original issue that was workarounded by #2716